transformer_lens/ActivationCache.py

```diff
@@ -43,6 +43,39 @@ def __init__(
         self.has_embed = "hook_embed" in self.cache_dict
         self.has_pos_embed = "hook_pos_embed" in self.cache_dict
 
+    @classmethod
+    def from_caches(cls, cache_list: List[ActivationCache], in_place: Bool = True) -> ActivationCache:
+        # Constructor that combines a list of caches into a big one
```

Comment:
Can you add a docstring? In particular, what `in_place` does should be documented